src/sparseml/pytorch/sparsification/quantization/quantize_qat_export.py

```diff
@@ -1065,6 +1080,9 @@ def _convert_quantizable_matmul_and_add(model: ModelProto):
         ):
             output_quantize_node = None
             output_dequantize_node = None
+        '''
```

Comment:
nit: Could we remove this commented out code?

Reply:
Removed